sdk/nodejs/appservice/zMixins_http.ts

```diff
@@ -51,10 +51,24 @@ export type HttpEventSubscriptionArgs = util.Overwrite<mod.CallbackFunctionAppAr
      * used.
      */
     location?: pulumi.Input<string>;
+
+    /**
+     * Defines the route template, controlling to which request URLs your function responds. The 
+     * default value if none is provided is <functionname>.
+     */
+    route?: string;
+
+    /**
+     * An array of the HTTP methods to which the function responds. If not specified, the function 
+     * responds to all HTTP methods.
+     */
+    methods?: string[];

```

Comment:
these should be Inputs.  `methods` should be `Input<Input<string>[]>`.

Reply:
Do I understand correctly that then `const bindings` become `Input`, the parent constructor has to change, and so on throughout the chain?